source/refunding_payments/index.html.md.erb

```diff
@@ -119,6 +119,22 @@ If the payment has reached your bank account, your PSP will take the refund amou
 - the next payment or payments you receive from any of your users, if your PSP is Stripe or ePDQ
 - your bank account, if your PSP is SmartPay or Worldpay
 
+If your PSP is Stripe, you can see when Stripe took the refund by making an API call to [get information about a payment](/reporting/#reporting-via-the-gov-uk-pay-api).
```

Comment:
Quick comment while I'm looking through the rest more carefully, should this link to advice on `get information about a refund` or is it intentionally payment oriented? 

We do seem to have sections about getting refunds https://docs.payments.service.gov.uk/refunding_payments/#checking-the-status-of-a-refund, I notice it doesn't mention getting a single refund (although this is possible through the API (https://govukpay-api-browser.cloudapps.digital/#get-a-payment-refund)

Reply:
Ah sorry - this is incorrect because I copied-and-pasted too quickly from another bit of the new docs.

I've updated now - could you check this is correct?